CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 
 ##### Minor
 
+* Allow to use proc for `:input_html` option when declaring filters [#5029][] by [@Fivell][]
```

Comment:
Change to: "Support `proc` as an `input_html` option value when declaring filters"